src/bosh.js

```diff
@@ -11,6 +11,16 @@ import core from 'core';
 
 const Strophe = core.Strophe;
 const $build = core.$build;
+let nodeXHR;
```

Comment:
Why can this not be moved to the `_newXHR` function?

Reply:
I honestly don't remember, I think I tried putting it there but the linter or even babel was complaining about having a require in the method.